admin_apps/vqr_app_mock.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+import time
+from typing import Literal
+import pandas as pd
+import streamlit as st
+from dataclasses import dataclass
+
+import streamlit as st
+from streamlit_monaco import st_monaco
```

Comment:
This `streamlit_monaco` will need to be added to the poetry env if we like it better than simple st.text_area